src/setup-binary.ts

```diff
@@ -32,46 +32,46 @@ export async function fetchBinary(versionSpec: string): Promise<string> {
 
   let binaryPath: string;
 
-  core.info(
-    `Finding an application version that matches version spec '${versionSpec}'.`
-  );
+  core.info(`Finding release that matches ${versionSpec}.`);
   let release = await hc.getRelease(BINARY_NAME, versionSpec, USER_AGENT);
 
-  const version = release.version;
-  core.info(`Found ` + BINARY_NAME + ` ${version}.`);
+  let version = release.version;
```

Comment:
having mutable variables (`let` instead of `const`) is generally frowned upon in JS/TS world, is it possible to use immutable approach here instead?

Reply:
Great catch - I've switched this back to `const`